pkg/utils/parsearguments.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+// StringMap implements the flag.Value interface and allows to parse stringified maps
+// in the form: "key1=val1,key2=val2".
+type StringMap map[string]string
```

Comment:
Nit. Could be defined as wrapper instead of alias, to prevent the need for subsequent casts, as in the [example here](https://pkg.go.dev/flag#example-Value)